integration-tests/capacity/transactions.test.ts

```diff
@@ -113,26 +114,21 @@ describe("Capacity Transactions", function () {
 
             // A registered provider with Capacity but no tokens associated with the
             // key should still be able to use polkadot UI to submit a capacity transaction.
-            // *All accounts will have at least an EXISTENTIAL_DEPOSIT = 1M.
-            // This test may still be valid if the txn cost is greater than 1M.
             it("pays for a transaction with available capacity and no free tokens", async function () {
                 let delegatorKeys = createKeys("delegatorKeys");
-                let _delegatorMsaId = await ExtrinsicHelper.createMsa(delegatorKeys);
+                await fundKeypair(devAccounts[0].keys, delegatorKeys, 100n * dollars);
```

Comment:
I don't understand the purpose of these changes in this test and above.  What was wrong with the test setup?

Reply:
The await does not belong there. It is doing nothing. It's broken.